documentation/meta/project_boards/issues.md

```diff
@@ -0,0 +1,49 @@
+# Issues project: Openverse Backlog
+
+The [Openverse Backlog](https://github.com/orgs/WordPress/projects/75) project
+board tracks all issues through their lifecycle, as they move from creation to
+completion. This board does not track any pull requests, so all workflows for
+this board are tied to events occurring for issues.
+
+## Event automations
+
+### Issue is created
+
+If a new issue is created, it is automatically added to the project board
```

Comment:
+If a new issue is created in the https://github.com/WordPress/openverse repository,

Reply:
Fixed in b54362b.